app/models/portfolio.rb

```diff
@@ -13,8 +13,9 @@
 class Portfolio < ApplicationRecord
   acts_as_tenant(:tenant)
 
-  validates_presence_of :name
-  validates_presence_of :description
+  validates :name, presence: true, :uniqueness => { :scope => :tenant_id }
+  validates :description, presence: true
```

Comment:
@jonnyfiveiq I have been meaning to ask if Portfolio description is meant to be a required field?

Reply:
I verified with @jonnyfiveiq that description is not a required field.  We should drop this validation.  

Also requires an update to the UI.  cc @lgalis @Hyperkid123

Reply:
As long as we get come reasonable error message back like: Name Foo is already taken. Or endpoint where we can check name availability there should be no problem with that.

Reply:
Agreed with UXD-Susan that all description fields will be optional.

thanks

Reply:
Removed validation on description being present.